### ISSUE TITLE
About the format of XML files

### ISSUE THREAD

**Issue by BangdongChen** [sohaib023/Table-Augmentation#1]
Hi, in line 98 of **_generate_samples.py_**, I found that **_doc_** is nothing after **_Document(xml_file)_**, is there something wrong with the format of the xml files? Can you give a xml file as an example?

**Comment by sohaib023** [comment]
There is a possibility of some problems with the code. (I'll advise you to use the version 1.0 of package truthpy, as there were some changes made to truthpy after it which were not backwards compatible). 
Apart from that, please share the XML file over here and I can have a look into it.

**Comment by sohaib023** [comment]
Looking through the code for [truthpy.Document](https://github.com/sohaib023/Truth-Py/blob/3f195b8f84534a2ec1b51ea66e59da94e3fede16/truthpy/Document.py#L9), I can see that it only reads the file if the path exists. In case if the path doesn't exist it silently creates an empty Document without throwing an error. I think that might be the cause for the issue you're facing.

**Comment by sohaib023** [comment]
I have attached an example ground truth file which can be used. As github did not allow me to upload XML so I changed the extension to txt. Do let me know if that clears up the issue.
[us-005_0_0.txt](https://github.com/sohaib023/Tab-Aug/files/7294998/us-005_0_0.txt)

**Comment by andyjiang1116** [comment]
i want to know how to generate the corresponding xml file if i use the pubtablenet dataset

**Comment by sohaib023** [comment]
@andyjpaddle I don't think you can convert the annotations for pubtablenet to the xml files required by this repo. I believe pubtablenet only has HTML annotations, i.e. there is no annotation regarding locations of the rows and columns in the image (which is needed for Tab-Aug). The only solution would be to manually annotate the images using [T-Truth](https://github.com/sohaib023/T-Truth).

**Comment by andyjiang1116** [comment]
most of table recognition datasets are labeled with text bbox and structure tokens, however TabAug need the cell bbox, in other words it can not be applied. i wonder is there a plan to support public dataset (eg, pubtabnet)?
